share/elections/2016-01.md

```diff
@@ -0,0 +1,40 @@
+# VoxPupuli 2016 First Elections
+
+This page contains the metadata about the first VoxPupuli Elections.
+
+# Scope
+
+Vote for the initial 5 members of the PMC
+
+# Election Officer
+
+The Election Officer should be appointed by the PMC. As there is none, the
+following people propose to run the first elections, and will step down from
+this role just after the election:
+
+* Julien Pivotto (roidelapluie@inuits.eu)
+  * Accepting candidatures
+  * Communication before the election
+* Daniele Sluijters
+  * Voting process
+  * Encouraging people to vote
+  * Communication for the election
+  * Announcement of the results
+
+According to our rules, the Election Officers can not run in the Election.
```

Comment:
yes, that is in the governance document https://github.com/voxpupuli/plumbing/blob/master/share/governance.md

> 1) Election Officer: The election officer can be on the PMC, as long as they are not running in the election they are officiating